rasa_nlu/utils/__init__.py

```diff
@@ -148,7 +148,7 @@ def module_path_from_object(o: Any) -> Text:
     return o.__class__.__module__ + "." + o.__class__.__name__
 
 
-def class_from_module_path(module_path: Text) -> Type[Any]:
+def class_from_module_path(module_path: Text) -> Type['Any']:
```

Comment:
I don't think these quotes are needed
```suggestion
def class_from_module_path(module_path: Text) -> Type[Any]:
```

Reply:
otherwise pycharm complained of the wrong class